drivers/wifi/nrf700x/osal/os_if/inc/osal_api.h

```diff
@@ -1319,4 +1319,17 @@ void wifi_nrf_osal_assert(struct wifi_nrf_osal_priv *opriv,
 			  int test,
 			  int val,
 			  char *msg);
+
+/**
+ * wifi_nrf_osal_str_len() - Gives the length of the string @str.
+ * @opriv: Pointer to the OSAL context returned by the @wifi_nrf_osal_init API.
+ * @str: Pointer to the memory location of the string.
+ *
+ * Calculates the length of the string pointed to by @str
+ *
+ * Return:
+ *              returns the number of bytes of the string @str.
+ */
+unsigned int wifi_nrf_osal_str_len(struct wifi_nrf_osal_priv *opriv,
```

Comment:
commit log:

```
drivers: wifi: Add strlen to the OSAL ops

<string.h> header file location is different between OSes, so,
add a OSAL op to hide this from OS agnostic code.
```

Reply:
If it's just a relative path, why can't we handle it by adding the path to includes in `CMakeLists.txt/Makefile` ?

Reply:
Commit log updated.
Inclusion of path in Makefile will not resolve inclusion of path to string.h header file in lnux case leads to other build issues.

Reply:
Same as https://github.com/nrfconnect/sdk-nrf/pull/12229#discussion_r1319747744